thirdeye/thirdeye-frontend/app/helpers/utils.js

```diff
@@ -46,6 +46,8 @@ export const eventColorMapping = {
   anomaly: 'teal'
 };
 
+export const dateFormatFull = 'ddd, MMM D YYYY, h:mm a';
```

Comment:
let's create a `constant.js` file to store all constants

Reply:
yes, something for a separate PR